lib/pbench/server/api/resources/graphql_api.py

```diff
@@ -1,6 +1,254 @@
 import requests
+import datetime
+import json
 from flask_restful import Resource, abort
-from flask import request, make_response
+from flask import request, make_response, jsonify
+from pbench.server.api.resources.auth import auth
+from pbench.server.api.resources.models import MetadataModel
+from pbench.server.api.resources.database import Database
+
+
+class UserMetadata(Resource):
```

Comment:
If I'm reading this right, this isn't even using the `graphql_schema` ??

It seems strange for it all to be bundled into the `graphql_api.py` module which is otherwise just the GraphQL server passthrough. (Which will presumably become obsolete when the dashboard actually switches to Server-based user and metadata APIs.)

Reply:
fixed